docs/gaia/gaia.rst

```diff
@@ -851,6 +852,45 @@ will be able to access to your shared table in a query.
   >>> Gaia.share_table_stop(table_name="user_<user_login_name>.my_table", group_name="my_group")
 
 
+
+---------------------------
+3. Datalink service (Public and Authenticated)
+---------------------------
+
+
+DataLink_ is a data access protocol compliant with the IVOA_ architecture that provides a linking mechanism between
+datasets offered by different services. In practice, it can be seen and used as a web service providing the list of additional
+data products available for each object outside the main catalogue(s). For more information about the products served via
+DataLink in the Gaia ESA Archive we recommend to read the Archive DataLink tutorials available at https://www.cosmos.esa.int/web/gaia-users/archive/datalink-products.
+
+The DataLink products are publicly accessible via Astroquery.Gaia using the *load_data* method. The following example shows how to retrieve the DataLink products associated to three sources in Gaia DR3:
```

Comment:
Do link to the API docs whenever possible. Also, moved the mention of the dictionary result from the dropped example below.

```suggestion
The DataLink products are publicly accessible via the `~astroquery.gaia.Gaia.load_data` method. 
The following example shows how to retrieve the DataLink products associated with three sources in Gaia DR3 as a python dictionary:
```

Reply:
We are happy with your changes.